LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-createHTMLDocument.html

```diff
@@ -56,7 +56,11 @@
 promise_test(t => {
   return new Promise(resolve => {
     trustedTypes.createPolicy("default",
-                              { createHTML: s => s + " [default]" });
+                              { createHTML: (s, _, sink) => {
+                                assert_equals(sink, 'Element innerHTML');
+                                return s + " [default]";
+                              }});
```

Comment:
nit: Indentation looks off might be worth fixing it as you're updating the code anwyay